internal/manifests/config/var.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+import "fmt"
+
+func fqdn(serviceName, namespace string) string {
+	return fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, namespace)
```

Comment:
I'm not sure, does this need to be configurable? https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/ provides a `--service-dns-domain` option (defaults to `cluster.local`).